tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -598,3 +598,51 @@ def validate_header_rpt_month_year(datafile, header, generate_error):
             field=None,
         )
     return is_valid, error
+
+def validate__WORK_ELIGIBLE_INDICATOR__HOH__AGE():
+    """If WORK_ELIGIBLE_INDICATOR == 11 and AGE < 19, then RELATIONSHIP_HOH != 1."""
+    # value is instance
+    def validate(instance):
+        WORK_ELIGIBLE_INDICATOR = (
+            instance["WORK_ELIGIBLE_INDICATOR"]
+            if type(instance) is dict
+            else getattr(instance, "WORK_ELIGIBLE_INDICATOR")
+        )
+        RELATIONSHIP_HOH = (
+            instance["RELATIONSHIP_HOH"]
+            if type(instance) is dict
+            else getattr(instance, "RELATIONSHIP_HOH")
+        )
+        RELATIONSHIP_HOH = int(RELATIONSHIP_HOH)
+
+        DOB = (
+            instance["DATE_OF_BIRTH"]
+            if type(instance) is dict
+            else getattr(instance, "DATE_OF_BIRTH")
+        )
+        DOB = str(DOB)
+        # 1974 01 14
+        DOB_datetime = datetime.datetime.strptime(DOB, '%Y%m%d')
```

Comment:
could one of these operations throw an exception if the dob is blank?

Reply:
Absolutely haha. I will make this safer!